ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/parsers/TomlParser.kt

```diff
@@ -40,18 +40,34 @@ public value class TomlParser(private val config: TomlConfig) {
         // here we always store the bucket of the latest created array of tables
         var latestCreatedBucket: TomlArrayOfTablesElement? = null
 
+        val comments: MutableList<String> = mutableListOf()
```

Comment:
We should be very careful here. Do you link your comments from previous lines to the next token?

Reply:
Yes, exactly